src/components/Login/Login.js

```diff
@@ -0,0 +1,103 @@
+import React, { Component } from 'react'
```

Comment:
This is not login :) try to find a better name for this file and this page. maybe `Menu`